Sources/LCG/LCG.swift

```diff
@@ -11,12 +11,18 @@ public struct Seed {
   
   public let seed: Int
   
-  public init(_ seed: Int) {
+  public init(seed: Int) {
     self.seed = (ensureBetween(Seed.min..<Seed.max) <| seed)
   }
   
-  public static func random() -> IO<Seed> {
-    return IO { Seed(Int(arc4random_uniform(UInt32(N)))) }
+  public static let random = IO {
+    (Seed.init(seed:) <<< Int.init <<< arc4random_uniform <<< UInt32.init) <| N
+  }
+}
+
+extension Seed: ExpressibleByIntegerLiteral {
```

Comment:
Seemed a good use-case for `ExpressibleByIntegerLiteral`. We don't wanna pass `Seed`s around by accident to `Int` receivers, but we do wanna make it easy to re-run a seed explicitly.